AccessibilitySnapshot.podspec

```diff
@@ -15,33 +15,30 @@ Pod::Spec.new do |s|
   s.default_subspecs = 'Core', 'SnapshotTesting'
 
   s.subspec 'Core' do |ss|
-    ss.source_files = 'AccessibilitySnapshot/Core/Classes/**/*.{swift,h,m}'
-    ss.public_header_files = [
-      'AccessibilitySnapshot/Core/Classes/UIAccessibilityStatusUtility.h',
-      'AccessibilitySnapshot/Core/Classes/UIView+DynamicTypeSnapshotting.h',
-    ]
+    ss.source_files = 'Sources/AccessibilitySnapshotCore/Classes/**/*.swift', 'Sources/AccessibilitySnapshotCore-ObjC/**/*.{h,m}'
+    ss.public_header_files = 'Sources/AccessibilitySnapshotCore-ObjC/include/*.h'
```

Comment:
Can all of the source files be in `Sources/AccessibilitySnapshot/Core/` (or `Sources/AccessibilitySnapshotCore`) and differentiate in SPM based on the file type? Feels really weird to separate these out when they're conceptually the same framework.

Reply:
SPM seems to struggle with mixed language targets/source file folders. This was one of the nicer ways I was able to get it all working!

It should be possible to do:
`Sources/AccessibilitySnapshot/Core/ObjC[/include]`
`Sources/AccessibilitySnapshot/Core/Swift`
If you think that's better?

Reply:
Hmm, that might work. Totally understand the limitation around mixed targets. I was suggesting breaking the strict target/directory pairing and explicitly defining the source file pattern, although it looks like SPM only supports an explicit source list (no glob matching). 😞

I think splitting in `/ObjC` and `/Swift` would be cleaner here though 👍

Reply:
Yeah the lack of glob matching is a bit sad 😭  I'll work on splitting it up like that!

Reply:
Should be resolved 👍  Sorry for the delay